src/app/(app)/alumni/page.tsx

```diff
@@ -94,23 +85,27 @@ export default function Alumni() {
                   </div>
                 ))}
               </div>
-            ) : (
+            ) : alumni?.length > 0 ? (
               <div className="grid grid-cols-2 gap-6 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-5">
-                {alumni?.sort((a, b) => a.name.localeCompare(b.name)) // Sort by name in alphabetical order
-                  .map((alum, index) => (
+                {alumni
+                  ?.sort((a, b) => a.name.localeCompare(b.name)) // Sort by name in alphabetical order
+                  .map((alum, index: number) => (
                     <Alum
                       key={index}
                       alumData={{
```

Comment:
use `alumniData` for all the alumni data, rename the `Alum` component to `AlumniCard`